src/app.ts

```diff
@@ -190,22 +191,25 @@ export function newApp(
       API_CLIENT
     );
 
+    // Create the Notification Service
+    // TODO: This call may throw an error
+    const NOTIFICATION_SERVICE = new NotificationService(
+      hubName,
+      endpointOrConnectionString,
+      ALLOW_MULTIPLE_SESSIONS
```

Comment:
avoid passing booleans as arguments, use an option object

Reply:
Fixed in 7f55174